src/java.naming/share/classes/com/sun/jndi/ldap/Connection.java

```diff
@@ -685,8 +685,6 @@ void cleanup(Control[] reqCtls, boolean notifyParent) {
                 } finally {
 
                     flushAndCloseOutputStream();
-                    // 8313657 socket is not closed until GC is run
```

Comment:
Would it be worth keeping a comment here about why `sock` is not closed, or at least mentioning 8362268?